app_dart/lib/src/service/luci_build_service.dart

```diff
@@ -510,19 +508,14 @@ class LuciBuildService {
     return false;
   }
 
-  bool _shouldRerunBuilder(LuciTask luciTask, String repo) {
-    if (luciTask.summaryMarkdown == null || !luciTask.builderName.contains('Mac')) {
+  /// Only blocking failures on framework TOT will be triggered.
```

Comment:
Should this get updated to match what the method is actually doing?

Reply:
Updated.